axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +199,62 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+
+class MoreGrofman(Player):
+    '''
+    Submitted to Axelrod's second tournament by Bernard Grofman.
+
+    This strategy has 3 phases:
+    1. First it cooperates on the first two rounds
+    2. For rounds 3-7 inclusive, it plays the same as the opponent's last move
+    3. Thereafter, it applies the following logic
+      - If its own previous move was C and the opponent has defected less than
```

Comment:
CI is failing because this isn't building in the documentation. Need a space:

```
    3. Thereafter, it applies the following logic

      - If its own previous move was C and the opponent has defected less than
```